snekbox/nsjail.py

```diff
@@ -35,11 +51,32 @@ def __init__(
         config_path: str = "./config/snekbox.cfg",
         max_output_size: int = 1_000_000,
         read_chunk_size: int = 10_000,
+        memfs_instance_size: int = 48 * Size.MB,
+        files_limit: int | None = 100,
+        files_timeout: float | None = 8,
+        files_pattern: str = "**/*",
     ):
+        """
+        Initialize NsJail.
+
+        Args:
+            nsjail_path: Path to the NsJail binary.
+            config_path: Path to the NsJail configuration file.
+            max_output_size: Maximum size of the output in bytes.
+            read_chunk_size: Size of the read buffer in bytes.
+            memfs_instance_size: Size of the tmpfs instance in bytes.
+            files_limit: Maximum number of output files to parse.
+            files_timeout: Maximum time in seconds to wait for output files to be read.
+            files_pattern: Pattern to match files to attach.
```

Comment:
Would it be possible to make the output directory part of the files_pattern? Would there be a problem with ambiguity in whether the directory should be created by NsJail or by the user?

Reply:
Well currently we have the glob start at the output folder, otherwise we'd have the output folder in the glob result and have to strip it for sending.

If we allow the output to be in the glob, the folder will still be created by NSJail so an end user will still need to change the code to have a different output folder name for example.

Reply:
I don't understand your answer, but I think it's because my question was unclear.

Currently, the output directory is hard-coded to `output`. I'm wondering if it'd be possible to make that configurable by allowing a user to specify the directory name in this `files_pattern` glob. However, I think this could result in ambiguity regarding what should be the path snekbox creates, and what should the end-user have to manually create. 

In any case, we can revisit it later. Not a blocker for the PR since I don't believe making this configurable in the future will be a breaking change.

Reply:
Ah I see what you mean. I think implicitly creating paths in the glob might be a bit complex (it also would allow some arbitrary traversal and other situations) I've made the name of `home` and `output` configurable now in [c3a43ac](https://github.com/python-discord/snekbox/pull/159/commits/c3a43ac3b254efc999727c9fd15aac64d3203162)

So to not use output, `memfs_output=""` can be set to set the output glob root to home, or any other path.